src/main/kotlin/no/nav/familie/tilbake/api/forvaltning/ForvaltningController.kt

```diff
@@ -212,6 +207,23 @@ class ForvaltningController(
             )
         }
     }
+
+    @Operation(summary = "Send siste tilstand for behandling til DVH")
+    @PostMapping(
+        path = ["/sendTilstandTilDVH"],
+        produces = [MediaType.APPLICATION_JSON_VALUE],
+    )
+    fun sendSisteTilstandForBehandlingerTilDVH(
+        @RequestBody behandlingIder: List<UUID>,
+    ) {
+        behandlingIder.forEach { behandlingID ->
+            val behandlingstilstand = behandlingTilstandService.hentBehandlingensTilstand(behandlingId = behandlingID)
+            kafkaProducer.sendSaksdata(
+                behandlingstilstand.behandlingUuid,
+                behandlingstilstand.copy(tekniskTidspunkt = OffsetDateTime.now(ZoneOffset.UTC)),
+            )
+        }
+    }
```

Comment:
Likte løsningen vi har i KS og BA, der vi oppretter en task for hver av behandlingene som sendes inn (synes det er litt mer ryddig og gir oversikt f.eks når kallet timer ut m/ 504 hvis man sender inn veldig mange behandlinger).

Er det mulig å gjøre det her og? Ser at vi allerede har SendSakshendelseTilDvhTask i tilbake, så kanskje det skal gå fint? Hvis det er for mye arbeid så funker nok dette helt kurant det også 😊

Reply:
klart det er mulig, men jeg og Halvor diskuterte akkurat det samme og tenkte at det var unødvendig med tasker :)
Er enig i at det er en fordel med rollback hvis noe timer ut.